.github/workflows/e2e.compatibility.yaml

```diff
@@ -18,98 +18,66 @@ name: Test Compatibility
-          ref: 5e4b16cbfc96f80bd86eb6a98ddcf60f0959c9b1
-
-      - name: Prepare envrionment
-        run: bash ${SCRIPTS_DIR}/pre.sh
-
-      - name: Install Minikube
-        run: bash ${SCRIPTS_DIR}/minikube.sh start
-
-      - name: Install Istio
-        run: bash ${SCRIPTS_DIR}/istio.sh --set profile=demo --set meshConfig.defaultConfig.envoyAccessLogService.address=skywalking-oap.istio-system:11800 --set meshConfig.enableEnvoyAccessLogService=true
-
-      - name: Install SkyWalking
-        run: |
-          cd chart
-          helm dep up skywalking
-          helm -n istio-system install skywalking skywalking \
-               --set fullnameOverride=skywalking \
-               --set elasticsearch.replicas=1 \
-               --set elasticsearch.minimumMasterNodes=1 \
-               --set oap.env.SW_ENVOY_METRIC_ALS_HTTP_ANALYSIS=k8s-mesh \
-               --set oap.envoy.als.enabled=true \
-               --set oap.replicas=1 \
-               --set ui.image.tag=${TAG//-es*} \
-               --set oap.image.tag=$TAG \
-               --set oap.storageType=elasticsearch
-          kubectl -n istio-system get pods
-
-          sleep 3
-          kubectl -n istio-system wait --for=condition=available deployments/skywalking-oap --timeout=1200s
-          kubectl get pods -A -o wide --show-labels
-          kubectl get services -A -o wide
-
-      - name: Deploy demo services
-        run: bash ${SCRIPTS_DIR}/demo.sh
-
-      - name: Cluster Info
-        if: ${{ failure() }}
-        run: |
-          df -h
-          minikube logs
-          minikube status
+          ref: 8436135dc919351a6d0ab42ff48c85e1b3104b53
+      - uses: actions/checkout@v2
+        with:
+          repository: apache/skywalking-satellite
+          submodules: true
+          path: satellite
+          ref: 4b97fd255b6459fe9d3fc4f88c0d8f682fc2a364
 
-      - name: Set up Minikube tunnel
-        run: |
-          mkdir /tmp/minikube-tunnel
-          minikube tunnel > /tmp/minikube-tunnel/a.log &
-          export POD_NAME=$(kubectl get pods -n istio-system -l "app=skywalking,release=skywalking,component=ui" -o jsonpath="{.items[0].metadata.name}")
-          echo $POD_NAME
-          kubectl -n istio-system port-forward $POD_NAME 8080:8080 > /tmp/minikube-tunnel/b.log &
+      - name: Build SkyWalking Satellite Docker Image
+        run: cd satellite && make docker && cd ..
 
-      - name: Run E2E test
+      - name: Load SkyWalking OAP And UI Image
         run: |
-          export GATEWAY_HOST=$(minikube ip)
-          export GATEWAY_PORT=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.spec.ports[?(@.name=="http2")].nodePort}')
-          export WEBAPP_HOST=127.0.0.1
-          export WEBAPP_PORT=8080
-
-          cd main && ./mvnw --batch-mode -f test/e2e/pom.xml -am -DfailIfNoTests=false verify -Dit.test=org.apache.skywalking.e2e.mesh.ALSE2E
+          docker pull $OAP_REPO:$OAP_TAG
+          docker pull $UI_REPO:$UI_TAG
```

Comment:
No need to do this

Reply:
We need to do this because the E2E kind environment should load these images from the local machine, if the images could not be found in the local machine the E2E will fail, it couldn't auto pull for now.

Reply:
e2e infra and KinD could automatically download images. What issue do you mean?

Reply:
> We need to do this because the E2E kind environment should load these images from the local machine, if the images could not be found in the local machine the E2E will fail, it couldn't auto pull for now.

The answer is not not to load the public images into the KinD because there is no need

Reply:
> e2e infra and KinD could automatically download images. What issue do you mean?

The kind import only support import the docker image from local machine, if local machine is not present the task will fail. So I need to pull the images to the local machine first, then the kind environment could import them.

Reply:
> > We need to do this because the E2E kind environment should load these images from the local machine, if the images could not be found in the local machine the E2E will fail, it couldn't auto pull for now.
> 
> 
> 
> The answer is not not to load the public images into the KinD because there is no need

Ok, you are right. It's don't need to pull first, because it's public. I will change it.

Reply:
No need to `load`, I think this is what we wanted to say actually.